CLAUDE.md

```
@@ -0,0 +1,46 @@
+# CLAUDE.md
+
+## Build & Test
+
+```bash
+go build -o wuphf ./cmd/wuphf        # build the binary
+bash scripts/test-go.sh               # full test suite (matches CI)
+bash scripts/test-go.sh ./internal/team  # single package
+```
+
+Web UI (in `web/`):
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)